README.md

```diff
@@ -1,181 +1,248 @@
 
 1. JavaScript tooling could be rewritten in a more performant language
 2. An integrated toolchain can tap into efficiencies that are not available to a disparate set of tools
 
-## Milestone
+## ⚡️ Quick Start
 
-This project currently has a fully working parser, a prototype for the linter as well as the minifier.
+The linter is ready to use. It comes with over 60 default rules and no configuration is required.
 
-The current objectives are:
+To start using, install [oxlint][npm-oxlint] or via `npx`:
 
-* Publish the linter as a product
-* Finish the minifier
+```bash
+npx oxlint@latest .
+```
 
-## Contributing
+To give you an idea of its capabilities, here’s an example from the [vscode] repository, which finishes linting 4000+ files in 0.5 seconds.
 
-This project is an invitation for you to come and learn Rust with us,
-any contributions to this project are appreciated.
+<p float="left" align="left">
+  <img src="https://raw.githubusercontent.com/Boshen/oxc-assets/main/linter-screenshot.png" width="60%">
+</p>
 
-To get started, check out some of the [good first issues](https://github.com/web-infra-dev/oxc/contribute) or ask us on [Discord][discord-url].
+## ⚡️ Performance
 
-If you are unable to contribute by code, you can still participate by:
+* The parser is currently the fastest ready-for-production Rust-based parser.
+* The linter is more than 50 times faster than [ESLint], and scales with the number of CPU cores.
 
-* star and watch this project
-* test the linter by running `npx oxlint@latest .` in your own projects
-* join us on [Discord](https://discord.gg/9uXCAwqQZW)
-* [follow me on twitter](https://twitter.com/boshen_c)
-* provide your wisdom in [discussions](https://github.com/web-infra-dev/oxc/discussions)
+<p float="left" align="middle">
+  <img src="https://raw.githubusercontent.com/Boshen/bench-javascript-parser-written-in-rust/main/bar-graph.svg" width="49%">
+  <img src="https://raw.githubusercontent.com/Boshen/bench-javascript-linter/main/bar-graph.svg" width="49%">
+</p>
 
-## Linter
+## ⌨️  Programming Usage
 
-The linter is fast to the extent that it feels broken.
+### Rust
 
-With 45 rules implemented, testing in the [VSCode](https://github.com/microsoft/vscode) repo on a Mac M2:
+Individual crates are published, you may use them to build your own JavaScript tools.
 
-```
-vscode  main ❯ npx oxlint@latest src
-Finished in 388ms on 3477 files with 45 rules using 8 threads.
-Found 798 warnings.
-```
+* The umbrella crate [oxc][docs-oxc-url] exports all public crates from this repository.
+* The AST and parser crates [oxc_ast][docs-ast-url] and [oxc_parser][docs-parser-url] are ready for use.
+* See `crates/*/examples` for example usage
 
-And also in a huge monorepo using Mac i7:
+The crates are lightweight and tuned for compilation speed. This is demonstrated by our [CI runs](https://github.com/web-infra-dev/oxc/actions/workflows/ci.yml?query=branch%3Amain),
```

Comment:
Compilation for who? You as a maintainer or for users ? This can be misleading because the crate does do compilation too.

Reply:
I changed this to

While Rust has gained a reputation for its comparatively slower compilation speed,
we have dedicated significant effort to fine-tune the Rust compilation speed.
Our aim is to minimize any impact on your development workflow,
ensuring that developing your own Oxc based tools remains a smooth and efficient experience.